public/js/discover/searchList.js

```diff
@@ -71,3 +76,29 @@ export function selectCard(id = null, scroll = false) {
     }
   }
 }
+
+/**
+ * Formats a raw phone number into a +# (###) ### - #### format.
+ * @param {string} number The phone number unformatted.
+ * @return {string} The formatted phone number.
+ */
+function formatPhoneNumber(number) {
+  if (number.length == 9) {
+    return (
+      '(' + number.substring(0, 3) + ') ' + number.substring(3, 6) + ' - ' + number.substring(0, 3)
+    );
+  } else if (number.length == 10) {
+    return (
+      '+' +
+      number.substring(0, 1) +
+      ' (' +
+      number.substring(1, 4) +
+      ') ' +
+      number.substring(4, 7) +
+      ' - ' +
+      number.substring(7, 10)
```

Comment:
Assign a variable to each of the different substrings of the phonenumber then use the variables when constructing the string. 

**Parts of a Phone Number**
![image](https://user-images.githubusercontent.com/26394550/87940799-2b4b8300-ca68-11ea-8b83-8c8e19afad3d.png)